src/plugins/data/public/search/search_interceptor.ts

```diff
@@ -52,6 +53,7 @@ export interface SearchInterceptorDeps {
   startServices: Promise<[CoreStart, any, unknown]>;
   toasts: ToastsSetup;
   usageCollector?: SearchUsageCollector;
+  uiActions?: UiActionsStart;
```

Comment:
would we be able to add ui actions without adding to the search interceptor like the query enhancements takes a dependency on it